api-report/odsp-driver-definitions.api.md

```diff
@@ -157,6 +157,8 @@ export enum OdspErrorType {
     fluidNotEnabled = "fluidNotEnabled",
     invalidFileNameError = "invalidFileNameError",
     outOfStorageError = "outOfStorageError",
+    // (undocumented)
+    serviceReadOnly = "serviceReadOnly",
```

Comment:
Consider "server" not "service"

The communication from ODSP in this case is specifically that you ended up on read only server. The reason why it is retryable is because the server is readonly not because the actual resource/document/data or entire 'service' is read only.

0.02, feel free to push back on that notion. 

The header appears to be X-SP-SERVERSTATE : ReadOnly=1

Reply:
I like service more than server. Server is a particular box to me, while whole service is down for this tenant (no matter what box one connects to).

Reply:
Makes sense, thanks for clarifying.